packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth/methods/call.ts

```diff
@@ -581,7 +580,8 @@ describe("Eth module", function () {
             );
           });
 
-          it("should throw an error because the key address used is invalid", async function () {
+          // TODO: https://github.com/NomicFoundation/edr/issues/104
+          it.skip("should throw an error because the key address used is invalid", async function () {
             await assertInvalidArgumentsError(
```

Comment:
@fvictorio I'm wondering if it'd be ok to add a flag to `assertInvalidArgumentsError` to only expect an error in EDR mode and not match the message so that we can have these enabled?

Reply:
Fixed by https://github.com/NomicFoundation/hardhat/pull/4666/commits/a24682b8fa5e9a45c0b2cfd65dd9431bd4b9b43f and https://github.com/NomicFoundation/hardhat/pull/4666/commits/728ee6af1efbe2766c4c23d15e8818dfbe396032